app/assets/stylesheets/forms.css.scss

```diff
@@ -0,0 +1,6 @@
+.govuk-checkboxes__label::after {
```

Comment:
Maybe there is a better way to fix this, or the cause for this is something I have not foreseen?